### PR TITLE
[CSLD-54] Add description of Peer Data

### PR DESCRIPTION
**Pull request by chshersh**
This PR add description of Peer Data. Though this PR also will change some `time-warp` layer documentation because after https://github.com/serokell/time-warp-nt/pull/54 merges to `master` some things change.

### CODE REVIEW

**Review by staenrey — APPROVED** [review]
Reviewed, changes added in a commit

**Comment by chshersh** [comment]
@staenrey Thanks! Could you please review once again? I've added new information.

File: _docs/technical/protocols/2017-01-01-application-level.md [hunk 1]
```diff
@@ -157,6 +157,15 @@ To distinguish from integers addition, concatenation is denoted here as `(++)`.
 | GtTag | `3` |
 | GtMsgContents | `3` |
 
+Hence `cardano` is shipped with Update system protocol, and message formats also
+can be changed. So nodes should be aware of the protocol the other peer is running.
```
Comment (rockbmb) [inline _docs/technical/protocols/2017-01-01-application-level.md]: "the other peer"? This is too specific, use the plural "peers".

File: _docs/technical/protocols/2017-01-13-binary.md [hunk 2]
```diff
@@ -170,6 +170,10 @@ ghci> hexEncode $ TinyVarInt (2^14 -1)
 "ff7f"
 ~~~
 
+Usually `ByteString`s in `cardano` encoded with length encoded as variable
```
Comment (rockbmb) [inline _docs/technical/protocols/2017-01-13-binary.md]: This sentence is a bit strange, cc @staenrey.

**Review by rockbmb — CHANGES_REQUESTED** [review]
Great work @ChShersh, many examples.
@staenrey please review the newer commit, afterwards I'll take another look and merge.

File: _docs/technical/protocols/2017-01-01-application-level.md (hunk 1 above)
Comment (staenrey) [reply _docs/technical/protocols/2017-01-01-application-level.md]: done

File: _docs/technical/protocols/2017-01-13-binary.md (hunk 2 above)
Comment (staenrey) [reply _docs/technical/protocols/2017-01-13-binary.md]: changed the sentence

**Review by rockbmb — APPROVED** [review]

**Pull request merged**